### PR TITLE
bump up submodule version to 2.9.2-SNAPSHOT to make it consistent with parent pom

### PR DESCRIPTION
**Pull request by seanzhougoogle**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1030,7 +1030,7 @@
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-javadoc-plugin</artifactId>
-            <version>2.9.1</version>
+            <version>2.9.2-SNAPSHOT</version>
```
Comment (yaojiefeng) [inline pom.xml:1033]: Revert this change

**Review by yaojiefeng — APPROVED** [review]

**Review by sechegaray — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (seanzhougoogle) [reply pom.xml:1033]: good catch

Comment (seanzhougoogle) [reply pom.xml:1033]: Sebastian merged the PR by mistake and addressed this issue in another PR : https://github.com/cdapio/hydrator-plugins/pull/1693

Comment (seanzhougoogle) [reply pom.xml:1033]: BTW, was looking into why this issue was not disclosed in my testing. it's because the plugin is configured in `release` profile,  maven won't complain if the profile is not used.
